pkg/manager/manager_tasks.go

```diff
@@ -506,7 +514,26 @@ func (s *singleton) Clean() {
 
 			wg.Add(1)
 
-			task := CleanTask{Scene: *scene}
+			task := CleanTask{Scene: scene}
+			go task.Start(&wg)
+			wg.Wait()
+		}
+
+		for _, gallery := range galleries {
+			s.Status.incrementProgress()
```

Comment:
s.Status.incrementProgress() 
shouldnt this be something like s.Status.setProgress(len(scenes)+j, total) ?
where j the for index for gallery
If you want to use increment shouldn't the Task total be set first and increment also used for scenes?

Reply:
I can do that. It was just done to not use `len`. The calls to `setProgress` set the total.

Reply:
i didn't notice that the status total was updated that way.
you can leave it as is then